README.md

```diff
@@ -174,3 +175,108 @@ Explanation of the switches:
 
 + `-e` <ELEMENT> : MD element type for search, e.g. _doi_ or _creator_
 + `-q` <QUERY> : MD content to start the search
+
+
+# Adding new parsers
+
+## Introduction
+
+The extractor tool uses python classes as parser modules that can process specific formats as described above.
+
+In order to add a new custom parser, you have to write a parser class and add it to the directory in the following location: `o2r-meta\parsers`.
+
+Note that future versions of o2r-meta might be able to register all parsers based on the files in the `parsers` directory. Until then it is necessary to register a new parser in the function `register_parsers` of `o2r-meta\extract\metaextract.py`.
```

Comment:
Suggest to not mention the actual function names here, leave the details to later.

> In order to add a new custom parser, you have to write a parser class and register it manually.
> To take advantage of non static methods, all parsers are designed as classes.

Then create an issue with the ideas of "The use of classes as parsers is currently still implemented in a rather naïve way." and "Also a governing class that provides inheritance features should be the goal of a better extractor.". The instructions should not be mixed with architectural plans.

Reply:
I know, it's conceptual - but since that feature is not present, I would really miss this hint in the readme. Someone who wants to add a parser and needs to learn about the extractor should be informed about this. The introduction chapter seems to be a good place for this.

Reply:
Ok, leave it in. Since this is also an idea for a new feature, there should still be an issue, you could then even link to it.

Reply:
Agreed. And I will revise the wording.